File: berkelium-java/src/main/java/org/berkelium/demo/runtime/Runtime1.java
// Copyright (c) 2013 The Berkelium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

package org.berkelium.demo.runtime;

import org.berkelium.api.BerkeliumFactory;
import org.berkelium.api.Runtime;

public class Runtime1 {
	@SuppressWarnings("unused")
	public static void main(String[] args) {
		/*GREEN*/// Create Runtime/*GREEN*/
		Runtime runtime = /*GREEN*/BerkeliumFactory.getInstance().createRuntime()/*GREEN*/;
		/*RED*/// TODO close runtime/*RED*/
	}
}

File: berkelium-cpp/src/demo/examples/runtime1.c
// Copyright (c) 2013 The Berkelium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

#include <berkelium.h>
#include <stdio.h>

int main(int argc, char* argv[])
{
	/*GREEN*/// Create Runtime/*GREEN*/
	BK_Runtime runtime = /*GREEN*/BK_BerkeliumFactory_createRuntime(/*HINT:BK_ENV*/NULL/*HINT*/)/*GREEN*/;
	BK_Runtime_free(/*HINT:BK_ENV*/NULL/*HINT*/, runtime);

	return 0;
}
